client/src/components/sources/sourceCard.tsx

```diff
@@ -88,6 +88,6 @@ export const SourceCard = ({
         onClose={() => setIsCollaboratorDialogOpen(false)}
         onUpdate={onUpdate}
       />
-    </React.Fragment>
+    </>
```

Comment:
Why did you remove that? I don't think I have any preference but this and `React.Fragment` are equivalent